lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+class Takeaway
+  attr_reader :list, :order, :quantity, :total
+
+  def initialize
+    @list = { "Dish 1" => 1.00,
+              "Dish 2" => 2.00,
+              "Dish 3" => 3.00 }
+    @order = []
+  end
+
+  def menu
+    list
+  end
+
+  def choose(dish, quantity=1)
+    quantity.times do
+      list.select { |k,v| order.push([k, v]) if k == dish }
+    end
+  end
+
+  def calculate(order)
+    calculate_quantities(order)
+    calculate_cost(order)
+  end
+
+  def place_order(order, quantity, total)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)